scripts/make_sourcetables_jointcal_hsc_test_data.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+# Convert all source catalogs in the hsc/ repo to sourceTable_visit using
+# default conversions.
+# Should be run from the testdata_jointcal base path
+# Note that --no-versions is specified because of the different version of the stack
+# used to generate the source tables (w_2020_22) vs the original data (w_2020_14).
+# This is necessary because we need obs_subaru from w_2020_18 or later
+# (including DM-24379) in order to do the sourceTable conversions, and from
+# w_2020_21 or later (including DM-25020) which includes the correct local background
+# measurement.  Finally, the default Source.yaml file needs to be modified to remove
+# the sky_source column which did not exist for the w_2020_14 RC2 reprocessing
+
+writeSourceTable.py ./hsc --output ./hsc --id filter=HSC-R^HSC-I --config doApplyExternalPhotoCalib=True --config doApplyExternalSkyWcs=True --no-versions
```

Comment:
Why are you using `doApplyExternal*` here? The data in `hsc/` does not have jointcal or fgcmcal run on it.

Reply:
I didn't come up with the terminology, in this case, and it is confusing.  This actually just means to recompute the "local" WCS and photoCalib values at the positions of the objects via a plugin, for processing that was done without that plugin.

Reply:
That is very unfortunate, because its the same name as the coadd/validate_drp options for using jointcal/fgcmcal.

Can you please pass it up the chain that these names are very confusing?